.eslintrc

```diff
@@ -8,12 +8,35 @@
     },
     "sourceType": "module"
   },
-  "plugins": [
-    "react"
-  ],
+  "plugins": ["react"],
   "rules": {
-    "comma-dangle": [2, "never"],
     "react/no-deprecated": 0,
-    "react/prefer-stateless-function": 0
+    "react/prefer-stateless-function": 0,
+    // TODO(jonathaningram): the following rules are turned off to quiet the
+    // linter during a transition between an old version of airbnb/javascript,
+    // introducing prettier and upgrading to the latest version of
+    // airbnb/javascript.
+    // The existence of these rules does not indicate that they should be
+    // ignored. It simply means that they should be slowly re-enabled as the
+    // codebase is updated.
+    "global-require": 0,
+    "import/imports-first": 0,
+    "import/newline-after-import": 0,
+    "import/no-extraneous-dependencies": 0,
+    "import/no-named-as-default": 0,
+    "import/no-unresolved": [2, { "ignore": ["skin"] }],
+    "import/prefer-default-export": 0,
+    "jsx-a11y/href-no-hash": 0,
+    "jsx-a11y/img-has-alt": 0,
+    "no-console": 0,
```

Comment:
some of these are fine during the transition but ones like this one `no-console` and ones like `prefer-const` `no-undef` seem like we should never have to turn these off.

Reply:
Yep we never want to turn those off. In other words, we should always prefer const, for example.

I have grouped all of the ones we need to revisit after my TODO comment.